tests/Tests/500__jupyterhub/test-jupyterlab-notebook.robot

```diff
@@ -53,3 +53,10 @@ Can Launch Python3 Smoke Test Notebook
   #Get the text of the last output cell
   ${output} =  Get Text  (//div[contains(@class,"jp-OutputArea-output")])[last()]
   Should Not Match  ${output}  ERROR*
+
+Verify Proper image is provided and server starts
+    [Documentation]    Verifies an image is provided by default and server starts
+    [Tags]    Sanity
+    ...       ODS-469
+    Launch JupyterHub Spawner From Dashboard
+    Spawn Notebook
```

Comment:
Based on ODS-469 and the corresponding JIRA issue (RHODS-1460) I think it is necessary to add some checks here like the spawning status (if it failed or not), the image tag, etc

Reply:
Updated

Reply:
Updated

Reply:
you should also check which image has been spanwed. This TC wants to check if the "default" image has been spawned, so it should check if a specific "default" images has been properly spawned